autopilot/autopilot.go

```diff
@@ -459,6 +407,88 @@ func (ap *Autopilot) blockUntilSynced(interrupt <-chan time.Time) (synced, block
 	}
 }
 
+func (ap *Autopilot) tick(forceScan bool, tickerFired chan struct{}) {
```

Comment:
`tick` is not a very descriptive name. `performMaintenance` or something like that would be better.